sar_pipeline/preparation/downloads/orbits.py

```diff
@@ -122,15 +126,29 @@ def download_orbits(
         # The logic in eof.download.main() tries CDSE first by default. set force_asf by source
         try:
             force_asf = source == "ASF"
-            orbit_paths = eof.download.main(
-                sentinel_file=scene_safe_file,
-                save_dir=save_dir,
-                cdse_user=cdse_user,
-                cdse_password=cdse_password,
-                force_asf=force_asf,
-                asf_user=asf_user,
-                asf_password=asf_password,
-            )
+            try:
```

Comment:
Not for this PR, but I'm wondering if eof.download.main could use a tidy up? It's a bit unintuitive that it takes both CDSE and ASF credentials at this level, although that is explained with the comment. For readability, I would find it more intuitive if it attempted to download with CDSE credentials at the time where you check `if source == "CDSE"` -- however, I haven't looked at the function in detail, so there may be good reasons for having the downloader handle that logic.